src/main/fc/config.c

```diff
@@ -242,6 +242,14 @@ void resetServoConfig(servoConfig_t *servoConfig)
 {
     servoConfig->servoCenterPulse = 1500;
     servoConfig->servoPwmRate = 50;
+
+    uint8_t servoIndex = 0;
+    for (int i = 0; i < USABLE_TIMER_CHANNEL_COUNT && i < MAX_SUPPORTED_SERVOS; i++) {
```

Comment:
Thanks good find. I thought I had fixed this one up previously - but must have missed the motorIndex one (done on PWM input correctly)